server/api/meeting/index.cjs

```diff
@@ -0,0 +1,118 @@
+const router = require('express').Router();
+const { Meeting, Message, Rating } = require('../../db/index.cjs');
+
+router.post('/', async (req, res, next) => {
+  try {
+    console.log(req.body.userId);
+    const newMeeting = await Meeting.create(req.body);
+    res.status(200).json(newMeeting);
+  } catch (err) {
+    next(err);
+  }
+});
+
+router.put('/:meetingId', async (req, res, next) => {
+  try {
+    const meeting = await Meeting.findByPk(req.params.meetingId);
+
+    if (meeting) {
+      res.json(await Meeting.update(req.body));
+    } else {
+      res.status(404).send('Meeting not found with id ' + req.params.meetingId);
+    }
+  } catch (err) {
+    next(err);
+  }
+});
+
+router.get('/:meetingId', async (req, res, next) => {
+  // REQUIRES ISADMIN MIDDLEWARE
+  try {
+    const meeting = await Meeting.findByPk(req.params.meetingId);
+
+    if (meeting) {
+      res.json(meeting);
+    } else {
+      res.status(404).send('Meeting not found with id ' + req.params.meetingId);
+    }
+  } catch (err) {
+    next(err);
+  }
+});
+
+router.delete('/:meetingId', async (req, res, next) => {
+  // REQUIRES ISADMIN MIDDLEWARE
+  try {
+    await Meeting.destroy({
+      where: {
+        id: req.params.meetingId,
+      },
+    });
+    res.end();
```

Comment:
What status code does res.end() send? Should we do a res.sendStatus(204) (i.e., 'No Data') instead?

Reply:
oh, I did res.end() because it'd never stop waiting if I dont res.send something and res.end() ends the api call without the need of sending anything

Reply:
and if I did "Should we do a res.sendStatus(204) (i.e., 'No Data') instead?" it'd always send that because nothing else is being sent